### PR TITLE
Rewrite of Working Together, formatting and small changes

### PR DESCRIPTION
**Pull request by dorianmarie**
- Formatting to be more easily scannable/readable
- Remove active/passive voice recommendations
- Add explanation for Ping pong pair programming
- Add KitKat for end of slogan (Have a break, have a...)
- Put links URLs at the bottom

### CODE REVIEW

**Comment by mike-burns** [comment]
Can you say more about removing the active voice recommendation?

(The` KitKat slogan is "Gimme a break", BTW.)

**Comment by dorianmarie** [comment]
> Can you say more about removing the active voice recommendation?

The guideline said:

(in Avoid)

> - Using passive voice (i.e. *"You will be asked some questions"*).
>   - It could sound like *"You will be ask some questions"*, which could lead to a potential misunderstanding of *"You will be asking some questions"*.

(in Prioritize)

> - Using active voice (i.e. *"They will ask you some questions"*) as suggested by [AAAL (American Association for Applied Linguistics)].

Which I find a buggy argument, passive voice doesn't have to be confusing: "Dorian is going to work on this task" is both useful to say, and explicit.

If we are in a meeting of more than two, you are not going to refer to one other person as You (therefor exclusing the other people), but as their first name or as their pronoun.

So I find the guideline weird, both active and passive voice have their place and their uses.

> (The` KitKat slogan is "Gimme a break", BTW.)

not according to the linked video: https://www.youtube.com/watch?v=fejBO1HZXVQ, it's "Have a break, have a KitKat"

[AAAL (American Association for Applied Linguistics)]: https://www.aaal.org/guidelines-for-communication-rights##

**Comment by mike-burns** [comment]
[Well I'll be](https://www.theguardian.com/media/2004/aug/03/advertising.uknews)! You're right about KitKat's slogan.

**Comment by dorianmarie** [comment]
Ok I'm really confused about passive vs active voices so I'm gonna do some research:

> In a sentence written in the active voice, the subject of sentence performs the action. In a sentence written in the passive voice, the subject receives the action.

So "Dorian is walking" (active voice) vs "A walk is performed by Dorian" (passive voice)

Yet, I don't see the issue with the passive voice.

File: working-together/README.md [hunk 1]
```diff
@@ -4,156 +4,209 @@ A guide for working well together.
...
-- Agree Working Environment (Two keyboards? Text editor and other tools)
-- Agree Pairing Style (Time-based? [Ping pong]?)
+- **Agree Scope**
+  - Pairing for two hours?
+  - Until the ticket is complete?
+  - Just get past the bug?
 
-**During:**
+- **Agree Physical/Virtual Location**
+  - "Will we both be comfortable here?"
```
Comment (louis-antonopoulos) [inline working-together/README.md:15]: For consistency with the other examples, should the quotation marks be removed here?
```suggestion
  - Will we both be comfortable here?
```

**Review by louis-antonopoulos — APPROVED** [review]
Changing the format to be more readable is very helpful!

File: working-together/README.md (hunk 1 above)
Comment (dorianmarie) [reply working-together/README.md]: Thanks ❤️

**Comment by dorianmarie** [comment]
@mike-burns would you be fine removing the active/passive voice bullet points?

**Review by smaboshe — APPROVED** [review]

**Pull request merged**